ingestion_tools/dataset_configs/template.yaml

```diff
@@ -1,137 +1,188 @@
-  reconstruction_software: REQUIRED
-  tomogram_version: 1
-  voxel_spacing: REQUIRED
-  authors: *dataset_authors
-
-standardization_config:
-  destination_prefix: '99999'
-  source_prefix: path/to/source
-  run_glob: frames/*
-  run_regex: /TS_\d\d$
-  run_name_regex: (.*)
-  frames_glob: frames/{run_name}_*.tif
-  frames_name_regex: (.*)
-  gain_glob: frames/CountRef.dm4
-  rawtlt_files:
-    - metadata/{run_name}.rawtlt
-    - metadata/{run_name}.xf
-    - metadata/mdocs_modified/{run_name}.mdoc
-  tiltseries_glob: metadata/{run_name}.st
-  ts_name_regex: (.*)\.rec
-  tomo_format: mrc
-  tomo_glob: tomograms/*.rec
-  tomo_regex: (.*)\.mrc
-  tomo_voxel_size: '13.48'
-  tomo_key_photo_glob: .*\.jpg
+annotations: OPTIONAL
+- metadata: REQUIRED
+    annotation_confidence: OPTIONAL
+      ground_truth_used: OPTIONAL, STRING
+      precision: OPTIONAL IF ground_truth_used ELSE null, FLOAT ([0, 100])
+      recall: OPTIONAL IF ground_truth_used ELSE null, FLOAT ([0, 100])
+    annotation_method: REQUIRED, STRING
+    annotation_object: REQUIRED
+      description: OPTIONAL, STRING
+      id: REQUIRED, STRING (GOID)
+      name: REQUIRED, STRING
+      state: OPTIONAL, STRING
+    annotation_publications: OPTIONAL, STRING
+    annotation_software: OPTIONAL BUT RECOMMENDED, STRING
+    authors:  OPTIONAL BUT RECOMMENDED
+    - ORCID: OPTIONAL, STRING (ORCID)
+      corresponding_author_status: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+      name: REQUIRED, STRING
+      primary_author_status: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+    dates: REQUIRED
+      deposition_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+      last_modified_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+      release_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+    ground_truth_status: REQUIRED, BOOLEAN (DEFAULT FALSE)
+    is_curator_recommended: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+    method_type: REQUIRED, ENUM (manual, hybrid, automated)
+    version: OPTIONAL, STRING
+  sources: REQUIRED
+  - binning: OPTIONAL, INTEGER (DEFAULT 1) (Point, OrientedPoint, InstanceSegmentation)
+    columns: OPTIONAL, STRING (DEFAULT xyz) (Point)
+    delimiter: OPTIONAL, STRING (DEFAULT ',') (Point)
+    file_format: REQUIRED, STRING
+    filter_value: OPTIONAL, STRING (OrientedPoint, InstanceSegmentation)
+    glob_string: REQUIRED, STRING (REGEX)
+    is_visualization_default: OPTIONAL, BOOLEAN (DEFUALT false)
+    mask_label: OPTIONAL, INTEGER (DEFAULT 1) (SemanticSegmentationMask)
+    order: OPTIONAL, STRING (OrientedPoint, InstanceSegmentation)
+    shape: OPTIONAL, ENUM (InstanceSegmentation, OrientedPoint, Point, SegmentationMask, SemanticSegmentationMask)
+dataset_keyphotos: OPTIONAL
+- sources: REQUIRED
+  - literal: REQUIRED
+      value: REQUIRED
+        snapshot: REQUIRED, STRING
+        thumbnail: REQUIRED, STRING
+datasets: REQUIRED
+- metadata: REQUIRED
+    authors: SEE annotations.authors
+    cell_component: OPTIONAL
+      id: OPTIONAL, STRING (GOID)
```

Comment:
Nit: as the other components that reference this might not all have Go IDs for their id, this could potentially be confusing.

Reply:
Reverted, thanks for catching that!